sites/public/pages/account/applications.tsx

```diff
@@ -1,83 +1,45 @@
-import React, { useEffect, useState, Fragment } from "react"
+import React, { useEffect, useState, Fragment, useContext } from "react"
 import Head from "next/head"
 import {
-  AppearanceBorderType,
-  AppearanceStyleType,
-  AppStatusItem,
-  Button,
+  ApiClientContext,
   DashBlock,
   DashBlocks,
   HeaderBadge,
   LinkButton,
   MetaTags,
-  Modal,
   RequireLogin,
   t,
+  UserContext,
 } from "@bloom-housing/ui-components"
 import Layout from "../../layouts/application"
-import moment from "moment"
-import { Application, ArcherListing } from "@bloom-housing/backend-core/types"
+import { PaginatedApplication } from "@bloom-housing/backend-core/types"
+import { AppStatusItemWrapper } from "./AppStatusItemWrapper"
 
 export default () => {
-  const [applications, setApplications] = useState([])
-  const [deletingApplication, setDeletingApplication] = useState(null)
-  const listing = Object.assign({}, ArcherListing)
+  const { applicationsService } = useContext(ApiClientContext)
+  const { profile } = useContext(UserContext)
+  const [applications, setApplications] = useState<PaginatedApplication>()
 
   useEffect(() => {
-    // applicationsService.list().then((apps) => {
-    //   setApplications(apps)
-    // })
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const listing = {} as any
-    const application = {} as Application
-    listing.applicationDueDate = moment().add(10, "days").format()
-    application.listing = listing
-    // TODO: Fix the types here (and probably this shouldn't come from the frontend anyway)
-    // application.updatedAt = moment().toDate()
-    setApplications([application])
-  }, [])
+    if (profile) {
+      applicationsService
+        .list({ userId: profile.id })
+        .then((apps) => {
+          setApplications(apps)
+        })
+        .catch((err) => console.error(`Error fetching applications: ${err}`))
```

Comment:
On this page it looks like any error condition will simply show that no applications are present. Could be frustrating if somebody's seeing that instead of at least a message that some kind of error occurred.

Reply:
Made this change, thanks!